### DISCUSSION TITLE
Observaciones de la primera versión final

### DISCUSSION THREAD

**Discussion by AbrilIchel** [grupoioa/gis_agm_series#79] [Ideas]
Observaciones
**1.	Selección de puntos**
a.	Sugerencia, al seleccionar los puntos de interés (que están resaltados), etiquetarlo por el nombre no por número de punto.
**2.	Data VIEW**
a.	En la sección de desplegar los datos en pantalla hay dos columnas, sugiero que la primer columna se refiera al nombre del mes y en los diarios el día juliano o día/mes,  igual que en el CSV. 
b.	Un encabezado que se refiera a la variable que se está viendo, suponiendo que el usuario quiere ver varias variables al mismo tiempo. 
c.	Close y refresh hacen la misma acción. 
**3.	Restore**
a.	No tiene acción.
**4.	Desplegar y descargar**
a.	Agregar las coordenadas en los títulos y leyendas 
b.	Puede ser confuso el 2018 en el despliegue, sugiero que se agregue solo el mes en el caso de los mensuales o el día juliano o día/mes en el caso de los diarios.
c.	Aunque la etiqueta tenga que ser un poco más larga de lo normal, creo que será más sencillo de visualizar y entender si se le agrega la palabra completa, por ejemplo, “WSM” >>  “Viento Mensual” o agregar una leyenda con la descripción nomenclatura. 
d.	Agregar unidades.
e.	Al eliminar todos los puntos,  vuelvo a seleccionar, la numeración continua. Sugiero que se reinicie la numeración. 
f.	Cuando modifíco las coordenadas manual, si se mueve en el mapa pero no despliega los datos. 
**5.	Sugerencias extras no indispensables pero dan calidad**
a.	Agregar eje de coordenadas en el mapa
b.	Marco en el mapa (así no se ve como una imagen recortada)
**6.	DUDA**
a.	El mapa (creo que es de google), ¿no tiene restricciones de uso?

**Comment by ma-robles** [comment]
Me parece que esto va mejor acá
